src/sha256t.rs

```diff
@@ -78,6 +78,28 @@ impl<T: Tag> HashTrait for Hash<T> {
     }
 }
 
+/// Macro used to define a newtype tagged hash.
+/// It creates two public types:
+/// - a sha246t::Tag struct,
+/// - a sha256t::Hash type alias.
+#[macro_export]
+macro_rules! sha256t_hash_newtype {
```

Comment:
May be `sha2561_newtype`, since `sha` already implies that it is a hash?

Reply:
I think the name is for symmetry with `hash_newtype`

Reply:
could be a tag... :) idk I think it's good to be inline with `hash_newtype`?